README.md

```diff
@@ -101,19 +101,17 @@ This part is relevant only if you want to develop the repository or build it on
 Only needed the first time:
 
 ```
+git clone https://github.com/Kixunil/cryptoanarchy-deb-repo-builder
+cd cryptoanarchy-deb-repo-builder
 sudo apt-get install cargo npm ruby-mustache
-# Required version >= 0.1.1
+# Requires the latest debcrafter version
 cargo install --git https://github.com/Kixunil/debcrafter
 PATH=$PATH:~/.cargo/bin
 cargo install cfg_me
-make build-dep
+make BUILD_DIR=$PWD/build build-dep
+make BUILD_DIR=$PWD/build
 ```
 
-Building
---------
-
-`make`
```

Comment:
This still needs to be separated somehow because the phrase above says it's only needed once. Removing the phrase can lead to a different kind of confusion. I suspect separate instructions will have to be written.

Reply:
427e926